Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -20,15 +20,25 @@ public class NetworkBehaviour : MonoBehaviour
         // this prevents recursion when SyncVar hook functions are called.
         bool m_SyncVarGuard;
 
+        /// <summary>True if the obejct is controlled by the client that owns it.</summary>
         public bool localPlayerAuthority { get { return netIdentity.localPlayerAuthority; } }
+        /// <summary>True if this object is running on the server, and has been spawned.</summary>
         public bool isServer { get { return netIdentity.isServer; } }
+        /// <summary>True if the object is running on a client.</summary>
         public bool isClient { get { return netIdentity.isClient; } }
+        /// <summary>True if the object is the one that represents the player on the local machine.</summary>
         public bool isLocalPlayer { get { return netIdentity.isLocalPlayer; } }
+        /// <summary>True if the obejct is only running on the server, and has been spawned.</summary>
         public bool isServerOnly { get { return isServer && !isClient; } }
+        /// <summary>True if the obejct is only runnign on the client.</summary>
         public bool isClientOnly { get { return isClient && !isServer; } }
+        /// <summary>True if this object is the authoritative version of the object..</summary>
```

Comment:
two dots at the end.

add a link here:
https://vis2k.github.io/Mirror/Concepts/Authority